### DISCUSSION TITLE
HTTPError: 401 Client Error with model-zoo

### DISCUSSION THREAD

**Discussion by RotoDrache** [Project-MONAI/MONAILabel#1232] [Q&A]
I apologize if this is a simple issue as I am new to MONAILabel. I am trying to use the **Use Case 2: Bundle with Customized Scripts for Renal Substructure Segmentation** in the QUICKSTART menu to segment the kidney.
I have installed monailabel in conda env. and have followed the steps in the quickstart but am getting the following error

File "/home/sam/anaconda3/envs/monailabel-weekly-env/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://api.github.com/repos/Project-MONAI/model-zoo/releases

Any help would be appreciated

**Comment by tangy5** [comment]
@RotoDrache , This is a Github authentication and rate limit error, it can happen when you try to access Bundle and Model-Zoo many times shortly. Can you try add this in your monailabel start_server command.

`--conf auth_token <Your Github Auth Token>
`

**Reply by RotoDrache** [reply]
Hi thank-you for your most prompt reply, its all good now!

**Reply by NovakBright** [reply]
Hello! I am new to MONAILabel and just start study Python.  I also have this error.  Can you teach me the detail code about <Your Github Auth Token>? I already create my github personal access token, but I don't know the detail code to run this command.
Thankfully!  I am grateful for your kind assistance.

**Reply by diazandr3s** [reply]
Hi @NovakBright,

Have you tried exporting the token like this? https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/monaibundle#step-4-start-monai-label-server-and-3d-slicer-to-annotate

Hope this helps,
